lib/airbrake-ruby/backtrace.rb

```diff
@@ -71,5 +81,16 @@ def stack_frame(match)
           function: match[:function] }
       end
     end
+
+    def self.match_frame(regexp, stackframe)
+      match = regexp.match(stackframe)
+      return match if match
+
+      unless (matched_frame = GENERIC_STACKFRAME_REGEXP.match(stackframe))
+        raise Airbrake::Error, "failed parsing '#{stackframe}'"
```

Comment:
Nitpick: "Failed parsing" assumes that there was some failure, but here we just "can't parse".